app/browser-sync.ts

```diff
@@ -93,6 +95,16 @@ export class TargetLocatorSync {
     exec(this.targetLocator.window(nameOrHandle));
   }
 
+  frame(nameOrElement: ElementFinderSync | string): void {
```

Comment:
Maybe it's better if this just accepts an element and the caller can select it however they'd like?